src/app/models/data/item.ts

```diff
@@ -17,9 +17,9 @@ export enum ItemId {
   CopperOre = 'copper-ore',
   CopperPlate = 'copper-plate',
   CrudeOil = 'crude-oil',
-  EfficiencyModule = 'efficiency-module',
-  EfficiencyModule2 = 'efficiency-module-2',
-  EfficiencyModule3 = 'efficiency-module-3',
+  EfficiencyModule = 'effectivity-module',
+  EfficiencyModule2 = 'effectivity-module-2',
```

Comment:
If I understand correctly, then the use of "hardcoded" variables does not allow working with modules added by mods?

Reply:
There are a lot of areas in the code that make assumptions that you are using the base mod. Rooting them out will take time, my first step is to rip out the oil and uranium utilities, issue #90, and replacing them with a generic solver that can handle complex recipes from mods.

Reply:
And certainly: PRs welcome 😄